community/kernel/src/main/java/org/neo4j/kernel/impl/store/id/BufferingIdGenerator.java

```diff
@@ -70,4 +70,14 @@ void maintenance()
     {
         buffer.maintenance();
     }
+
+    @Override
+    public void close()
+    {
+        if ( buffer != null )
+        {
+            buffer.maintenance();
```

Comment:
Rather we should call `buffer.close()` since it has one and does additional closy stuff there.